app/views/home-new.njk

```diff
@@ -0,0 +1,77 @@
+{% extends '_layout.njk' %}
+{% set mainClasses = "govuk-main-wrapper--auto-spacing" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block content %}
+  <div class="govuk-width-container">
+    <main class="govuk-main-wrapper">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="govuk-heading-l">Progressive reductions: calculate how BPS and delinked payments will reduce</h1>
+
+          <p class="govuk-body">Payments from the Basic Payment Scheme (BPS) in England are reducing each year from 2021 until the end of the 2023 scheme year.</p>
+          <p class="govuk-body">Delinked payments will replace BPS payments from 2024 and will reduce each year until payments finish at the end of 2027.</p>
+          <p class="govuk-body">These reductions to BPS and delinked payments are known as progressive reductions.</p>
+          <p class="govuk-body">Use the BPS calculator to get an estimate of how progressive reductions could affect your BPS payments from 2021 to 2023.</p>
+
+          {{ govukButton({
+            classes: "govuk-!-margin-top-2 govuk-!-margin-bottom-8",
+            href: "/value-calculator",
```

Comment:
Page name should reflect page content, I would consider something like `bps-calculator`.

Reply:
renamed all references from value-calculator to bps-calculator